changelog/10384-bugfix.md

```diff
@@ -0,0 +1 @@
+Validate regular expressions in nlu training data configuration.
```

Comment:
@carlad I think we forgot the changelog in the original pull request. This changelog entry would have been already in 2.8.x and main branches and a release would delete those changelog files, and insert their contents into `CHANGELOG.mdx`. What would be the best course of action to fix this? Maybe merging changelogs and doing the release afterwards?